pages/services/nifi/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: layout.pug
-navigationTitle:  Apache NiFi
-title: DC/OS Apache NiFi
+navigationTitle:
+title: NiFi
```

Comment:
would this fix the name here as well? https://docs.mesosphere.com/services/

Should just be `NiFi` on https://docs.mesosphere.com/services/